onboarding/matplot.py

```diff
@@ -1,7 +1,34 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
```

Comment:
For any new dependencies, please add them to the `requirements.txt` so they can be easily installed